lib/src/commonMain/kotlin/sovran.kotlin/Store.kt

```diff
@@ -226,21 +219,18 @@ class Store {
     // Removes any expired subscribers.
     private suspend fun clean() {
         sovranScope.launch(syncQueue) {
-            subscriptions.removeAll {
-                it.owner.get() == null
-            }
+            subscriptions.clear()
```

Comment:
here we only clear subscriptions that are no longer valid, instead of clear all subscriptions.

Reply:
would be put back when we have the owner weakreference sorted.